### PR TITLE
Add whitelisted 'postmessage' to valid redirect URIs

### PR DESCRIPTION
**Pull request by quartzmo**
closes: #92

### CODE REVIEW

File: lib/googleauth/user_authorizer.rb [hunk 1]
```diff
@@ -206,13 +206,7 @@ def get_and_store_credentials_from_code options = {}
       #  Unique ID of the user for loading/storing credentials.
       def revoke_authorization user_id
         credentials = get_credentials user_id
-        if credentials
-          begin
-            @token_store.delete user_id
-          ensure
-            credentials.revoke!
-          end
-        end
```
Comment (quartzmo) [inline lib/googleauth/user_authorizer.rb]: I refactored the lines above to a single line to solve a Rubocop complaint:

`lib/googleauth/user_authorizer.rb:54:5: C: Metrics/ClassLength: Class has too many lines. [113/110]`

Comment (dazuma) [reply lib/googleauth/user_authorizer.rb]: Honestly, we should prefer updating the rubocop config over doing this.

Comment (quartzmo) [reply lib/googleauth/user_authorizer.rb]: Totally agree, happy to do that.

Comment (quartzmo) [reply lib/googleauth/user_authorizer.rb]: Increased ClassLength to 200, removed previous exclusion.

**Pull request merged**